composer.json

```diff
@@ -20,6 +20,7 @@
   },
   "require": {
     "php": ">=5.3.3",
-    "apache/log4php": "2.3.0"
+    "apache/log4php": "2.3.0",
```

Comment:
Is log4php really a "require"? Maybe a "suggest"?
https://getcomposer.org/doc/04-schema.md#suggest